# goal
float64 distance
float64 angle
---
# result
float64 final_x
float64 final_y
float64 final_theta
---
# feedback
float64 cur_x
float64 cur_y
float64 cur_theta
